test/2_management_api/notifications/boundary_event_notifications.js

```diff
@@ -0,0 +1,332 @@
+'use strict';
+
+const should = require('should');
+const uuid = require('node-uuid');
+
+const {ProcessInstanceHandler, TestFixtureProvider} = require('../../../dist/commonjs/test_setup');
+
+describe('Management API:   Receive global BoundaryEvent Notifications', () => {
+
+  let eventAggregator;
+  let processInstanceHandler;
+  let testFixtureProvider;
+
+  let defaultIdentity;
+
+  const processModelId = 'test_management_api_boundaryevent';
+
+  let correlationId;
+
+  let boundaryEventToFinish;
+
+  const noopCallback = () => {};
+
+  const boundaryEventWaitingMessagePath = 'boundary_event_reached';
+  const boundaryEventFinishedMessagePath = 'boundary_event_finished';
+  const sampleBoundaryEventMessage = {
+    correlationId: uuid.v4(),
+    processModelId: 'processModelId',
+    processInstanceId: uuid.v4(),
+    flowNodeId: 'BoundaryEvent_1',
+    flowNodeInstanceId: uuid.v4(),
+    processInstanceOwner: undefined, // Can only be set after the TestFixtureProvider was initialized.
+    payload: {},
+  };
+
+  before(async () => {
+    testFixtureProvider = new TestFixtureProvider();
+    await testFixtureProvider.initializeAndStart();
+    defaultIdentity = testFixtureProvider.identities.defaultUser;
+
+    sampleBoundaryEventMessage.processInstanceOwner = defaultIdentity;
+
+    const processModelsToImport = [processModelId];
+    await testFixtureProvider.importProcessFiles(processModelsToImport);
+
+    eventAggregator = await testFixtureProvider.resolveAsync('EventAggregator');
+    processInstanceHandler = new ProcessInstanceHandler(testFixtureProvider);
+  });
+
+  after(async () => {
+    await testFixtureProvider.tearDown();
+  });
+
+  it('should send a notification via socket when BoundaryEvent is suspended', async () => {
```

Comment:
```suggestion
  it('should send a notification via socket when a BoundaryEvent is reached', async () => {
```

BoundaryEvents are never suspended. They only get triggered, if their triggering event occurs.

See my other remarks (here)[https://github.com/process-engine/process_engine_contracts/pull/115], I'm not certain that publishing two notifications for a BoundaryEvent is the way to go anyway.

Reply:
👍 You're right, it might be enough to send the finished event